src/main/resources/swagger.api/mod-entities-links.yaml

```diff
@@ -290,11 +290,20 @@ components:
         linkingRuleId:
           type: integer
           description: ID of linking rule by which link should be created
+        status:
+          type: string
+          readOnly: true
+          description: Status of the link
+        errorCause:
+          type: string
+          readOnly: true
+          description: Link update error cause (if present)
       required:
         - authorityId
         - authorityNaturalId
         - instanceId
         - linkingRuleId
+        - status
```

Comment:
Why status is required? We expect it to have in request body?

Reply:
If field is `readOnly` then `required` applies only to response according to https://swagger.io/docs/specification/data-models/data-types/#readonly-writeonly